diff --git a/scripts/generate-schema.ts b/scripts/generate-schema.ts
index 9cd8cccb7f5521c27890aade6d45b2774786185a..ec58886a041751c6426dd7ef99dcdf42155e7f5b 100644
--- a/scripts/generate-schema.ts
+++ b/scripts/generate-schema.ts
@@ -705,7 +705,7 @@ function generateMeta(): MetaResponse {
   }
 
   for (const filePath of loaderFiles) {
-    const relativePath = path.relative(srcDir, filePath);
+    const relativePath = path.relative(srcDir, filePath).replace(/\\/g, "/");
     const loaderKey = `${SITE_NAMESPACE}/${relativePath}`;
 
     try {
@@ -834,7 +834,7 @@ function generateMeta(): MetaResponse {
   }
 
   for (const filePath of sectionFiles) {
-    const relativePath = path.relative(srcDir, filePath);
+    const relativePath = path.relative(srcDir, filePath).replace(/\\/g, "/");
     const blockKey = `${SITE_NAMESPACE}/${relativePath}`;
 
     try {
@@ -921,7 +921,7 @@ function generateMeta(): MetaResponse {
     }
 
     for (const filePath of appFiles) {
-      const relativePath = path.relative(srcDir, filePath);
+      const relativePath = path.relative(srcDir, filePath).replace(/\\/g, "/");
       const blockKey = `${SITE_NAMESPACE}/${relativePath}`;
 
       if (
